src/PublicAssets.php

```diff
@@ -38,7 +24,7 @@ public static function enqueue_js() {
 		wp_register_script(
 			'main',
 			$theme_dir . '/assets/build/index.js',
-			[ 'jquery' ],
+			[],
```

Comment:
Hey @lithrel @mleray if we leave this intact for a while would that include WP jquery version? Just thinking about letting this for now to give a bit more time on child themes for migration.

Reply:
right, it does force the inclusion of jQuery 3.6.1 coming from `wp-includes/js`.

Reply:
Even when we remove this it's still included from `wp-includes/js` 🤔

Reply:
I don't know how/if we can solve that

Reply:
Looks like maybe we could use WordPress' [wp_deregister_script](https://developer.wordpress.org/reference/functions/wp_deregister_script/) function

Reply:
> Even when we remove this it's still included from `wp-includes/js` thinking

It's not a cache issue ? works on local for me, might be another block/plugin loading it on it's side depending on your page ?

Reply:
It works as expected for me on local now that I've added [this commit](https://github.com/greenpeace/planet4-master-theme/pull/1852/commits/42bddbaf25f4460f9670a0d0905395a45ed71843)

Reply:
Of course if we want to keep it for a while for NROs as suggested by @comzeradd, we would need to remove that

Reply:
Yes, let's revert this last commit and re-add the change above in the diff to make sure jquery is still enqueued in the frontend. We can then merge the PR and re-test this when we want to completely remove the library.